website/content/docs/enterprise/license/faq.mdx

```diff
@@ -7,7 +7,8 @@ description: An overview of license.
 # License FAQ
 
 This FAQ section is for the license changes introduced in Vault Enterprise 1.11.
```

Comment:
Should this be updated? It seems like we're adding docs surrounding new behaviour in 1.12, so perhaps it should say "1.11 and 1.12"?

Reply:
Good catch. This has been sort of a living document and accounted for license changes since 1.8. It might make sense to call out all versions mentioned so that folks can quickly identify whether or not they may be affected. What do you think?

Reply:
If it's more of a living document, it might make sense to just remove the mention of versions?

Maybe something like "This FAQ section is for license changes and updates introduced for Vault Enterprise"

Reply:
Love it, thanks for the suggestion.